src/content/docs/en/guides/styling.mdx

```diff
@@ -520,11 +520,11 @@ Svelte in Astro also works exactly as expected: [Svelte Styling Docs][svelte-sty
 
 ## Markdown Styling
 
-Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts).
+Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts). Please note that layout styling is not the same as the rendered Markdown content styling.
```

Comment:
```suggestion
Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts), but different methods will have different styling effects on your page.
```
original sentence unclear

Reply:
I'm fine with the change as is, but the original change proposed was meant to emphasize the difference between the Markdown Layout component styling and the styling for rendered markdown content.

Let me know what you think about this:

```suggestion
Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts), but may not apply to styling for rendered markdown content.
```